### DISCUSSION TITLE
dmripep dwi output space

### DISCUSSION THREAD

**Discussion by halilaziz** [nipreps/dmriprep#199] [General]
Hello folks,
 I am running dmriprep via docker. 
 It is finishing succesfully but when I check the output space, there are only anat, figures. and logs folders. There is no dwi folder. Is there any solution for it?
 Thank you
 
 You may find my docker command below:
 docker run --rm -it -v /analysis/halilaziz/bids_raw:/data -v /analysis/halilaziz/dmriprep:/output -v /usr/local/local/opt/freesurfer6/license.txt:/usr/local/local/opt/freesurfer6/license.txt nipreps/dmriprep:latest /data /output/ participant  --participant-label 12193  --nthreads 8 --omp-nthreads 8 --mem_mb 16000  --low-mem --notrack --fs-license-file /usr/local/local/opt/freesurfer6/license.txt  --output-space dwi  --skip-bids-validation

**Comment by maryjis** [comment]
Yes, I have the similar problem. With the latest docker image, run gets completed with no errors, but I have only two files in dwi output (brain mask and b0 ref), no one  dwi derivitives

**Comment by halilaziz** [comment]
Hi Maria,

I recommend you use just the FSL edyy_openmp command for the preprocessing.
I think dmriprep is not so useful.

Best
Halil Aziz



--

*Halil Aziz VELIOGLU, PhD*

*Postdoctoral Research Trainee*
Feinstein Institute for Medical Research, Center for Psychiatric
Neuroscience

*Assistant Professor*
Istanbul Medipol University, functional Imaging and Cognitive-Affective
Neuroscience Lab (fINCAN)


Maria Zubrikhina ***@***.***>, 7 Haz 2023 Çar, 10:26
tarihinde şunu yazdı:

**Comment by arokem** [comment]
dmriprep is still pre-alpha software, so it still does not produce useful outputs. I think that's something we should add to the README and documentation of the software. I'll make a PR.
